src/components/pages/BookPage/starRate.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+function CreateStars({ rate }) {
+  let stars = [];
+  let firstDigit = rate - parseInt(rate);
+  let secondDigit = firstDigit.toString().split('.')[1];
+  let createStars = () => {
+    for (let i = 1; i <= 5; i++) {
+      if (i <= rate) {
+        stars.push(<icon className="fa fa-star text-yellow-300 mr-1 "></icon>);
+      } else {
+        if (secondDigit === 5) {
```

Comment:
Did you tested with something like 3.75 stars I don't think it going to give the right results and same with 3.6 3.8 3.7 3.9 ?

Reply:
mohamed told me to do only half star

Reply:
@ZeenaKareemAlhello yeah but now you are only taking the half
I mean the 3.6 , 3.7 , 3.8 , 3.9 should treat as 3.5 or at least 4